conda_smithy/lint_recipe.py

```diff
@@ -164,20 +200,25 @@ def lintify_meta_yaml(
     lint_license_should_not_have_license(about_section, lints)
 
     # 11: There should be one empty line at the end of the file.
-    lint_should_be_empty_line(meta_fname, lints)
+    lint_should_be_empty_line(recipe_fname, lints)
 
     # 12: License family must be valid (conda-build checks for that)
-    try:
-        ensure_valid_license_family(meta)
-    except RuntimeError as e:
-        lints.append(str(e))
+    if not is_rattler_build:
+        try:
+            ensure_valid_license_family(meta)
```

Comment:
can you add a comment that `rattler-build` enforces spdx and therefore doesn't need a `license_family`?